test/lightning/invocation_test.exs

```diff
@@ -226,40 +233,38 @@ defmodule Lightning.InvocationTest do
     end
 
     test "update_run/2 with invalid data returns error changeset" do
-      run = run_fixture()
+      run = insert(:run)
 
       assert {:error, %Ecto.Changeset{}} =
                Invocation.update_run(run, @invalid_attrs)
-
-      assert run == Invocation.get_run!(run.id)
     end
 
     test "delete_run/1 deletes the run" do
-      run = run_fixture()
+      run = insert(:run)
       assert {:ok, %Run{}} = Invocation.delete_run(run)
       assert_raise Ecto.NoResultsError, fn -> Invocation.get_run!(run.id) end
     end
 
     test "change_run/1 returns a run changeset" do
-      run = run_fixture()
+      run = insert(:run)
       assert %Ecto.Changeset{} = Invocation.change_run(run)
     end
 
     test "list_work_orders_for_project/1 returns workorders ordered by last run finished at desc, with nulls first" do
-      fixture = workflow_job_fixture(workflow_name: "chw-help")
-      job = fixture.job
-      trigger = fixture.trigger
-
-      dataclip = dataclip_fixture()
+      project = insert(:project)
+      workflow = insert(:workflow, project: project)
+      job = insert(:job, workflow: workflow)
+      trigger = insert(:trigger, workflow: workflow)
 
-      reason = reason_fixture(dataclip_id: dataclip.id, trigger_id: trigger.id)
+      dataclip = insert(:dataclip)
 
-      workflow = job.workflow
+      reason =
+        insert(:reason, type: :webhook, dataclip: dataclip, trigger: trigger)
 
-      wo_one = work_order_fixture(workflow_id: workflow.id)
-      wo_four = work_order_fixture(workflow_id: workflow.id)
-      wo_two = work_order_fixture(workflow_id: workflow.id)
-      wo_three = work_order_fixture(workflow_id: workflow.id)
+      wo_one = insert(:workorder, reason: reason, workflow: workflow)
+      wo_four = insert(:workorder, reason: reason, workflow: workflow)
+      wo_two = insert(:workorder, reason: reason, workflow: workflow)
+      wo_three = insert(:workorder, reason: reason, workflow: workflow)
```

Comment:
A little tip here, ExMachina has a `insert_list/3` function, this could be written like this:

```elixir
[wo_one, wo_four, wo_two, wo_three] = 
  insert_list(4, :workorder, reason: reason, workflow: workflow)
```